### ISSUE TITLE
Inverter for 36v battery 

### ISSUE THREAD

**Issue by marine1988** [syssi/esphome-soyosource-gtn-virtual-meter#165]
Hello o have a bunch of 36v lithio batteries and wnat to make usefulk on my solar system. I saw this inverter grid tie with limiter that are perfect except for the fact that is wired limited an my inverter is away from the total of my grid. I have home assistant and shelly 3pm values can I use it to limit my inverter? Sorry I'm noob thank you

**Comment by card917** [comment]
yes you can do it if your consumption sensor exposed to ha. you can put the sensor name in powermeter0 section in the code. make sure that the unit is in W. mine is in kW then i have to put a filter to multiply by 1000. then it works just fine.

**Comment by syssi** [comment]
Please keep in mind the Soyosource can be used as photovoltaics inverter **or** battery inverter only. You cannot use the inverter to charge your batteries!

**Comment by marine1988** [comment]
Yes i know i bought 1 mppt charge to charge batteries i bought soyosource with wifi version seller say it only can have wifi or screen not both so I prefed wifi now i installed the code in esp home logs show is working but i have no information about the inverter whet do I need to do now?

**Comment by marine1988** [comment]
@syssi can I send a pm to you please I need some help with my configuration 
![Uploading 20240226_125351.jpg…]()

**Comment by syssi** [comment]
Could you try to upload your image another time?

**Comment by marine1988** [comment]
yes i can
![20240226_125351](https://github.com/syssi/esphome-soyosource-gtn-virtual-meter/assets/6524068/51139c1d-826a-415e-af46-596a4ed75d1c)

**Comment by syssi** [comment]
Do you still have questions? :-) Are you happy with your setup and can we close this issue?
